Snail/Observable.swift

```diff
@@ -188,15 +188,15 @@ public class Observable<T>: ObservableType {
         scheduler.start()
 
         var next: T?
-        scheduler.observable.subscribe(onNext: {
+        _ = scheduler.observable.subscribe(onNext: {
```

Comment:
I think we should add this to the subscribers array right? This way we remove this subscription when `deinit`